codecov.yml

```diff
@@ -0,0 +1,6 @@
+codecov:
+  max_report_age: off
+
+comment:
+  layout: "reach, diff, flags, files"
+  after_n_builds: 14
```

Comment:
Indeed, if you upload more reports, you will then have comment updates again.
If you upload less reports, you will notice that no comment is coming in.

I personally always was a bit confused when the first comment reduces the coverage and then edits and edits come in that increase the coverage, so I thought I put this in alongside.
Actually I've read it in a newsletter and it was my main intent to set that property. :-D

I don't think you can calculate it from the matrix unless you add a parser that parses the workflow file and interprets it to calculate how many iterations it will be.

Reply:
I added one commit that now calculates the value from the build matrix.
But I also marked the PR as "Draft" as it now depends on #1630.